data/presets/building/construction.json

```diff
@@ -1,5 +1,11 @@
 {
     "icon": "maki-building",
+    "fields": [
+        "name",
+        "address",
+        "opening_date",
+        "check_date"
```

Comment:
Less important, but I am curious in hearing opinions if it would makes sense to also add default building fields: `building/levels `and `height` here. I suppose the usage could be unclear, given that it could be interpreted as either planned or current. I didn't manage to find any docs on this, but personally I think it would be beneficial to use these in similar context as `check_date` (to update them during the construction) in order to track the current state and progress.